docs/base-chain/network-information/troubleshooting-transactions.mdx

```diff
@@ -0,0 +1,110 @@
+
+<Note>
+Base has a [minimum base fee](/base-chain/network-information/network-fees#minimum-base-fee). Transactions with `maxFeePerGas` below this value will never be included, since the base fee cannot drop below the minimum.
+</Note>
+
+### Priority Fee Too Low
+
+During periods of high demand, transactions compete for block space through priority fees. If your priority fee is too low relative to other transactions, yours may be delayed.
+
+**Solution**: Most users simply wait for congestion to subside. For time-sensitive transactions, use `eth_maxPriorityFeePerGas` to get a priority fee estimate that can outbid enough recent transactions to be included.
+
+<Note>
+If [DA throttling](https://docs.optimism.io/chain-operators/guides/configuration/batcher#batcher-sequencer-throttling) is currently in effect, there's no RPC endpoint that calculates priority fee estimates with throttling in mind. During DA throttling, even transactions with high priority fees may be delayed as the sequencer limits L2 transactions to manage its L1 data availability throughput.
+</Note>
+
+### Nonce Gap
+
+If you have a pending transaction with nonce N, all transactions with nonce N+1 or higher will queue behind it, regardless of their fees.
+
+**Solution**: Either wait for the pending transaction to be included, or replace it by submitting a new transaction with the same nonce and a higher fee (at least 10% higher `maxPriorityFeePerGas` and `maxFeePerGas`).
+
+### Nonce Too Low
+
+If you submit a transaction with a nonce that has already been used, it will be rejected.
+
+**Solution**: Query your current nonce using `eth_getTransactionCount` with the `pending` tag to get the next available nonce.
+
+## Transaction Rejected
+
+### Gas Limit Exceeds Maximum
+
+Base enforces a [per-transaction gas maximum](/base-chain/network-information/block-building#per-transaction-gas-maximum) of **25,000,000 gas**. Transactions specifying a higher gas limit are rejected by the mempool before inclusion.
+
+**Error**: `exceeds maximum per-transaction gas limit`
+
+**Solution**: Reduce the gas limit to 25,000,000 or below. If your transaction genuinely requires more gas, you'll need to break it into multiple transactions.
+
+## Transaction Included But Failed
+
+If your transaction was included in a block but shows a failed status:
+
+### Out of Gas
+
+The transaction ran out of gas during execution.
+
+**Solution**: Increase the gas limit. Use `eth_estimateGas` to get a gas estimate, then add a buffer (e.g., 20%) to account for variability.
+
+### Reverted by Contract
+
+The contract execution encountered a revert condition.
+
+**Solution**: Check the transaction on [Basescan](https://basescan.org) to see the revert reason. Common causes include failed require statements, arithmetic errors, or invalid state transitions.
```

Comment:
Should we explicitly identify Basescan or use the more generic explorer.base.org, which can be rerouted in the case of a service disruption.